docs/user-guide/upgrade-zos-system.md

```diff
@@ -0,0 +1,90 @@
+
+## Updating the cross memory server load modules
+
+Zowe releases contain two load modules, `ZWESIS00` for the cross memory server and `ZWESAUX` for the auxiliary server.  There are delivered in a PDSE `SZWEAUTH` that is created by the installation process.  
+
+If this PDSE is the same one used by the `ZWESISTC` proclib that starts the cross memory server, then because the installation replaces the data set and its contents no action is required.  This is the recommended approach to configure a Zowe environment.
+
+If you have copied the `SZWEAUTH` members to another PDSE that you are using as a runtime load module, then you should re-copy the updated members to the runtime location.  
+
+## Updating the system and security configuration
+
+The JCL member `ZWESECUR` delivered in the PDS member `SZWESAMP` contains the TSO commands used to configure a z/OS environment for launching Zowe.  
+
+The contents of `ZWESECUR` do not usually get updated between Zowe releases, so during an upgrade you should not need to re-run the JCL.  If there are additions then the HOLDDATA for the SMP/E release and the [release notes](../getting-started/summaryofchanges.md) will describe the changes.  
+
+## Upgrading the keystore directory
+
+When the previous release of Zowe was configured a keystore directory would have been created.  This would either contain the Zowe certificate, or else reference a SAF keyring that contains the Zowe certificate, see [Configuring Zowe certificates](./configure-certificates.md).  The USS keystore directory is created using the script `<RUNTIME_DIR>/bin/zowe-setup-certificates.sh` that is delivered with the new Zowe release, and creates the USS file `<KEYSTORE_DIRECTORY/zowe-setup-certificates.env` containing key value parameters used by the Zowe runtime to locate its certificate.  
+
+Typically the `<KEYSTORE_DIRECTORY>` is forward compatible can be used when moving forward to a new Zowe release.  There are situations when new functionality is introduced into a Zowe release when new key values pairs may be introduced to the `<KEYSTORE_DIRECTORY>/zowe-setup-certificates.env` file, in which case the new release will describe the new functionality in its HOLDDATA (for SMP/E) or release notes.  If this occurs it will be necessary to create a new `KEYSTORE_DIRECTORY` in order to exploit the new functionality.  This was the case when Zowe 1.13 introduced support for storing certificates in a SAF keyring.  
```

Comment:
keyring beta was added in 1.15, not 1.13

Reply:
Tks